org.gnupg.GPA.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.gnupg.GPA
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: gpa
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  #- --filesystem=~/.gnupg:rw
+  #- --filesystem=xdg-run/gnupg:rw
+  - --filesystem=host
+  - --filesystem=/tmp
+
+rename-desktop-file: gpa.desktop
+rename-icon: gpa
+
+modules:
+
+  - shared-modules/gtk2/gtk2.json
+
+  - name: gpa
+    cleanup:
+      - "*.a"
+      - "*.la"
+      - "/cache"
+      - "/include"
+      - "/lib/*.a"
+      - "/lib/pkgconfig"
+      - "/man"
+      - "/share/aclocal"
+      - "/share/devhelp"
+      - "/share/gtk-doc"
+      - "/share/man"
+      - "/share/pkgconfig"
+    buildsystem: autotools
+    post-install:
+      - install -Dm644 ${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/gpa.png
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: git
+        url: git://git.gnupg.org/gpa.git
```

Comment:
Would be probably better to use https://dev.gnupg.org/source/gpa.git as the URL here instead.